app/templates/admin/createEvent.html

```diff
@@ -1,9 +1,11 @@
 {% set title = "Create Single Event" %}
 
 {% set isNewEvent = 'create' in request.path %}
-{% if isNewEvent %}
-  {% if template.tag == 'single-program' %}             
-    {% set page_title = 'Create Event for ' + eventData["program"].programName %}
+{% if isNewEvent %}          
+    {% if eventData["program"].programName == 'CELTS-Sponsored Event' %}
```

Comment:
Make sure the indentation matches the rest of the `if` statement.

Reply:
Worked On Them : )